### DISCUSSION TITLE
Relations in breadcrumbs

### DISCUSSION THREAD

**Discussion by linaspasv** [filamentphp/filament#6096] [Help]
Hello,

I have a pretty complex relationships and would want to show relations in the breadcrumbs so user would have easier time to not get lost in them. I have tried to look for something like this in the documentation but could not find anything close.

e.g. lets say I have "Shops -> Products" relation where each Shop has many Products and when user views a particular Product I would want to augment breadcrumb to also include a name of the Shop (maybe with a link to view / edit shop when clicked).

Please advice.

Thanks.

**Comment by OPNG23** [comment]
up^^

**Comment by danharrin** [comment]
You can do this with nested resources in v4
